boards/shields/st_b_cams_imx_mb1854/st_b_cams_imx_mb1854.overlay

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#include <zephyr/dt-bindings/gpio/raspberrypi-csi-22pins-connector.h>
```

Comment:
It seems like `22pin` can expand to either "22 pins" (normal sentence) or "22-pin" (abused as an unit), but it can help avoiding confusion to keep it the same everywhere. Feel free to offer a push to the OV7670 shield  too if preferring 'pins'.